Source/Redemption/Private/RedemptionPlayer.cpp

```diff
@@ -131,6 +131,17 @@ void ARedemptionPlayer::Tick(float DeltaTime)
 
 	// Update the HUD with the new stamina value
 	UpdateStaminaWidget(Stamina / MaxStamina);
+	// Show/hide the stamina widget based on sprinting and stamina value
+	if (bIsSprinting || Stamina != MaxStamina)
```

Comment:
This means the widget will always be visible when the player is not at full stamina, regardless of whether they are sprinting or not.
Am i correct?

Reply:
Thats right, it's visible when its recharging and when being used.